pkg/fanal/analyzer/os/release/testdata/echo

```diff
@@ -0,0 +1,2 @@
+ID=echo
+VERSION_ID=1
```

Comment:
In echo.go, the scanner always returns true for IsSupportedVersion() with the comment "Echo is a rolling distro". However, the test data shows VERSION_ID=1, which suggests Echo might have versions. Can you please elaborate on that?

Reply:
We actually didn't want to put the `VERSION_ID` field. But because of [this](https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/analyzer/os/release/release.go#L79) line of code, we must put a value there, we chose `1`, but there is no specific reason for it.